doc/email-features.md

```diff
@@ -0,0 +1,87 @@
+# Email Features
+
+## Setup
+
+### Postmark
+
+Atomic CRM uses [Postmark](https://postmarkapp.com/) to receive inbound emails. You can use an existing Postmark account or [create a new one](https://account.postmarkapp.com/sign_up). The free tier allows you to send or receive up to 100 emails per month.
+
+To enable inbound email features, you need to create a webhook in Postmark.
+
+1. Go to your [Postmark account](https://account.postmarkapp.com/)
+2. Select your **server** (e.g. _My First Server_)
+3. Select the **Inbound Stream** (e.g. _Default Inbound Stream_)
+4. Go to the **Setup Instructions** tab
+5. Get your **server’s inbound email address** (xxxxxxx@inbound.postmarkapp.com)
+6. Set your **server’s inbound webhook URL**: `https://<user>:<password>@<project_id>.supabase.co/functions/v1/postmark` (e.g. https://user:pwd@xxxxxxx.supabase.co/functions/v1/postmark)
+7. **Save changes**. That's it for the Postmark setup!
+
+> **Note**: The webhook URL includes 3 important pieces:
+> - `<user>` and `<password>` are credentials you need to choose to secure the webhook. Please note them down as you will need them in the next step.
+> - `<project_id>` is the Supabase project's `SUPABASE_PROJECT_ID`
+
+> **Note**: The server’s inbound email address can be customized by registering a domain to your Postmark account.
+
+### Supabase
+
+Atomic CRM uses a Supabase Edge Function to handle the webhook and process the received emails. You need to configure your Supabase project to handle the incoming requests.
+
+In a terminal, after you have [linked your Supabase project](./supabase-configuration.md#using-an-existing-remote-supabase-instance), run the following commands:
+
+```sh
+npx supabase secrets set POSTMARK_WEBHOOK_USER=<user>
```

Comment:
**issue:** This should be explained in supabase configuration as well as it can break `make start-prod`

Reply:
This won't break `make start-prod` but I can reference this doc in the supabase configuration doc as well